src/lib/common/PathUtilities.h

```diff
@@ -25,6 +25,7 @@ class PathUtilities
     static std::string basename(const std::string& path);
     static std::string concat(const std::string& left, const std::string& right);
 
+    static const char DefaultDelimiter;
```

Comment:
This can be removed when the single location where it's used starts using the `concat` function.